web/client-api/src/main/java/io/deephaven/web/client/api/i18n/JsTimeZone.java

```diff
@@ -14,120 +14,538 @@
+        addJsonMapping("KR", constants.asiaSeoul());
+        addJsonMapping("KST", constants.asiaSeoul());
+        addJsonMapping("HK", constants.asiaHongKong());
+        addJsonMapping("HKT", constants.asiaHongKong());
+        addJsonMapping("SG", constants.asiaSingapore());
+        addJsonMapping("SGT", constants.asiaSingapore());
+        addJsonMapping("Asia/Kolkata", constants.asiaCalcutta());
+        addJsonMapping("IN", constants.asiaCalcutta());
+        addJsonMapping("IST", constants.asiaCalcutta());
+        addJsonMapping("TW", constants.asiaTaipei());
+        addJsonMapping("NL", constants.europeAmsterdam());
+        addJsonMapping("CE", constants.europeBerlin());
+        addJsonMapping("CES", constants.europeBerlin());
+        addJsonMapping("CEST", constants.europeBerlin());
+        addJsonMapping("LON", constants.europeLondon());
+        addJsonMapping("BST", constants.europeLondon());
+        addJsonMapping("CH", constants.europeZurich());
+        addJsonMapping("BT", constants.americaSaoPaulo());
+        addJsonMapping("BRST", constants.americaSaoPaulo());
+        addJsonMapping("BRT", constants.americaSaoPaulo());
+        addJsonMapping("NF", constants.americaStJohns());
+        addJsonMapping("NST", constants.americaStJohns());
+        addJsonMapping("NDT", constants.americaStJohns());
+        addJsonMapping("AT", constants.americaHalifax());
+        addJsonMapping("AST", constants.americaHalifax());
+        addJsonMapping("ADT", constants.americaHalifax());
+        addJsonMapping("ET", constants.americaNewYork());
+        addJsonMapping("EST", constants.americaNewYork());
+        addJsonMapping("EDT", constants.americaNewYork());
+        addJsonMapping("MN", constants.americaChicago());
+        addJsonMapping("CT", constants.americaChicago());
+        addJsonMapping("CST", constants.americaChicago());
+        addJsonMapping("CDT", constants.americaChicago());
+        addJsonMapping("MT", constants.americaDenver());
+        addJsonMapping("MST", constants.americaDenver());
+        addJsonMapping("MDT", constants.americaDenver());
+        addJsonMapping("PT", constants.americaLosAngeles());
+        addJsonMapping("PST", constants.americaLosAngeles());
+        addJsonMapping("PDT", constants.americaLosAngeles());
+        addJsonMapping("AL", constants.americaAnchorage());
+        addJsonMapping("AKST", constants.americaAnchorage());
+        addJsonMapping("AKDT", constants.americaAnchorage());
+        addJsonMapping("HI", constants.pacificHonolulu());
+        addJsonMapping("HST", constants.pacificHonolulu());
+        addJsonMapping("HDT", constants.pacificHonolulu());
+        addJsonMapping("SYD", constants.australiaSydney());
+        addJsonMapping("AEST", constants.australiaSydney());
+        addJsonMapping("AEDT", constants.australiaSydney());
+        addJsonMapping("MOS", constants.europeMoscow());
+        addJsonMapping("SHG", constants.asiaShanghai());
```

Comment:
for each of these, rather than re-reading the string into a json object, what about something like

```
addMapping("SHG", "Asia/Shanghai")
```

with

```
    private static void addMapping(String key, String existingKey) {
        JsTimeZone existing = timeZones.get(existingKey);
        if (existing == null) {
            throw new IllegalArgumentException("Existing key didn't exist " + existingKey);
        }

        addMapping(key, existing);
    }
```

Reply:
I like